types/types.go

```diff
@@ -131,6 +132,13 @@ type natsOutputConfig struct {
 	MinimumPriority string
 }
 
+type stanOutputConfig struct {
+	NatsHostPort    string
```

Comment:
```suggestion
	HostPort    string
```

For keeping a coherence with others